Assets/Python/GreatPeople.py

```diff
@@ -666,15 +679,16 @@ def setup():
 		"Yosa Buson", # 18th
 		iIndustrial,
 		"Katsushika Hokusai", # 19th
-		"Utagawa Hiroshige", # 
+		"Utagawa Hiroshige", # 19th
+		"fHiguchi Ichiyou", # 19th
```

Comment:
Japanese names in the mod follow Western conventions (first name, last name), so it should be `fIchiyou Higuchi`.